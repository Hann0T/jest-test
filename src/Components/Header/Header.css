.header {
  box-sizing: border-box;
  margin-bottom: 3em;
  padding: 1em 0;
  background-color: #0a146e;
  color: white;
}
