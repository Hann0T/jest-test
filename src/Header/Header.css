.header {
  background-color: #0a146e;
  color: white;
}
