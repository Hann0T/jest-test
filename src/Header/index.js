import React from 'react';

import './Header.css';

const Header = () => {
  return (
    <header className='header'>
      <div className='container'>
        <h1>Convert US Dollars to Peruvian Soles</h1>
      </div>
    </header>
  );
};

export { Header };
