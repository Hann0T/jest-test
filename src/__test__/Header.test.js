import React from 'react';
import renderer from 'react-test-renderer';

import { Header } from '../Header';

test("The header component didn't change", () => {
  const header = renderer.create(<Header />);

  expect(header.toJSON()).toMatchSnapshot();
});
