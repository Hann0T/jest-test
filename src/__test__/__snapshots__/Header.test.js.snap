// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The header component didn't change 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <h1>
      Convert US Dollars to Peruvian Soles
    </h1>
  </div>
</header>
`;
